### DISCUSSION TITLE
Not able to run deepstream-tests

### DISCUSSION THREAD

**Discussion by PugalendhiMurugan** [OE4T/meta-tegra-community#121] [Q&A]
Hi ,
I'm not able to run deepstream-tests.
Since deepstream path is not available on the compiled image, Is the any package dependencies?
DEEPSTREAM_PATH="/opt/nvidia/deepstream/deepstream-7.0"
Thanks,
Pugalendhi M

**Comment by madisongh** [comment]
Can you provide a bit more detail on
* which branch you're using
* what image you're building
* what you did to include the deepstream packages in your build?

If you're just looking for an easy way to run some of the tests, I'd suggest starting with our [demo distro](https://github.com/OE4T/tegra-demo-distro).  Check out the branch you want to use (scarthgap is probably a good choice, if you are looking for DeepStream 7.0), set it up for the Jetson you want to build for, then add the following to your `local.conf`:
```
CORE_IMAGE_EXTRA_INSTALL = "deepstream-tests"
```
and build the `demo-image-sato` image.   When the image boots up, you should find things under `/opt/nvidia/deepstream`, and you can just use the `run-deepstream-tests` script to run a collection of the DS samples. You should do that on from a terminal window on the graphics display.
